packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -78,10 +78,7 @@ export declare class VideoPlayer extends SharedObject<VideoPlayerEvents> {
 
   /**
    * Boolean value indicating if the player should correct audio pitch when the playback speed changes.
-   * > On web, changing this property is not supported, the player will always correct the pitch.
    * @default true
-   * @platform android
-   * @platform ios
```

Comment:
Is this change intentional?

Reply:
Yes, initially the preserve pitch didn't work on web, but then it turned out it's possible to apply it. It's supported now and the docs weren't updated apparently.